build-system/tasks/storybook/amp-env/main.js

```diff
@@ -20,7 +20,11 @@ module.exports = {
     '../../../../extensions/**/*.*/storybook/*.amp.js',
   ],
   addons: [
-    '@storybook/addon-a11y',
+    // TODO(alanorozco): AMP previews are loaded inside an iframe, so the a11y
+    // addon is not able to inspect the tree inside it. Its results are incorrect,
+    // since it checks this wrapper iframe. Enable this eventually, once we find a
+    // way to inspect the iframe's tree.
+    // '@storybook/addon-a11y',
```

Comment:
@caroqliu since you're our a11y expert.

I'm disabling the a11y panel on the AMP Storybook, since it was already useless. We should probably set up a task to try to make it work, unless we think it's enough to rely on the Preact's component analysis.

Reply:
The Preact component analysis should suffice for _most_ cases, but components that render with `detached` mode should have some analysis enabled in the AMP layer if possible. 

Can we render our AMP examples [inline](https://cloudfour.com/thinks/storybook-5-docs-inline-html/#optional-inline-by-default) instead of inside an iframe?